cinema/models.py

```diff
@@ -84,40 +84,24 @@
     row = models.IntegerField()
     seat = models.IntegerField()
 
+    @staticmethod
+    def validate_position(value: int, max_value: int, field_name: str, error_to_raise):
+        if not (1 <= value <= max_value):
+            raise error_to_raise({
+                field_name: f"{field_name} must be in the range [1, {max_value}]"
+            })
+
     def clean(self):
-        for ticket_attr_value, ticket_attr_name, cinema_hall_attr_name in [
-            (self.row, "row", "rows"),
-            (self.seat, "seat", "seats_in_row"),
-        ]:
-            count_attrs = getattr(
-                self.movie_session.cinema_hall, cinema_hall_attr_name
-            )
-            if not (1 <= ticket_attr_value <= count_attrs):
-                raise ValidationError(
-                    {
-                        ticket_attr_name: f"{ticket_attr_name} "
-                        f"number must be in available range: "
-                        f"(1, {cinema_hall_attr_name}): "
-                        f"(1, {count_attrs})"
-                    }
-                )
-
-    def save(
-        self,
-        force_insert=False,
-        force_update=False,
-        using=None,
-        update_fields=None,
-    ):
+        cinema_hall = self.movie_session.cinema_hall
+        Ticket.validate_position(self.row, cinema_hall.rows, "row", ValidationError)
+        Ticket.validate_position(self.seat, cinema_hall.seats_in_row, "seat", ValidationError)
```

Comment:
The `clean` method is correctly implemented to validate the row and seat positions using the `validate_position` method. This ensures that the ticket's position is within the valid range of the cinema hall's rows and seats.